src/temgymbasic/gui.py

```diff
@@ -1205,7 +1275,7 @@ def build(self) -> Self:
         self.ysize.lineEdit.textChanged.connect(self.set_shape)
         self.pixelsizeslider, _ = labelled_slider(
             self.detector.pixel_size, 0.001, 0.03, name="Pixel size",
-            insert_into=hbox, decimals=3,
+            insert_into=hbox, decimals=5,
```

Comment:
I'm afraid the decimals argument only selects between a float/int slider now, needs to be changed to `bool`. Before it selected the decimal stepsize but now that's abstracted to `superqt`.